core/types/market.go

```diff
@@ -999,6 +1000,12 @@ func MarketFromProto(mkt *vegapb.Market) (*Market, error) {
 	if mkt.ParentMarketId != nil {
 		parent = *mkt.ParentMarketId
 	}
+	var ls *LiquidationStrategy
+	if mkt.LiquidationStrategy != nil {
+		if ls, err = LiquidationStrategyFromProto(mkt.LiquidationStrategy); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Will nil be good on a protocol upgrade? Should have default here?

Reply:
I'm ensuring nil will be acceptable. It kind of has to be until the actual functionality is implemented anyways. This is just part of the feature in this PR (to prevent a massive PR that will be a pain to maintain/review/merge)